apps/admin-gui/src/app/shared/components/dialogs/attribute-export-dialog/attribute-export-dialog.component.html

```diff
@@ -0,0 +1,6 @@
+<h1 mat-dialog-title>Export attribute definition</h1>
```

Comment:
missing translation

Reply:
Actually, I forgot to remove this dialog, since it is not used at all. So I have removed it.